src/dev/build/lib/__tests__/exec.js

```diff
@@ -26,7 +26,7 @@ describe('dev/build/lib/exec', () => {
     sinon.assert.calledWithExactly(onLogLine, sinon.match(`$ ${process.execPath}`));
 
     // log output of the process
-    sinon.assert.calledWithExactly(onLogLine, sinon.match(/debg\s+hi/));
+    sinon.assert.calledWithExactly(onLogLine, sinon.match(/debg.+hi/));
```

Comment:
__note:__ that's to workaround bug mentioned above (basically skipping ansi color symbols after `debg` that aren't whitespaces).